client/lib/plans/features-list.js

```diff
@@ -76,10 +85,14 @@ export const FEATURES_LIST = {
 			i18n.translate( '{{a}}All Premium features{{/a}}', {
 				components: {
 					a: (
-						<a
+						<ExternalLinkWithTracking
 							href="https://jetpack.com/features/comparison"
 							target="_blank"
-							rel="noopener noreferrer"
+							tracksEventName="calypso_plan_link_click"
+							tracksEventProps={ {
+								link_location: 'plan_features_list_item',
+								plan_slug: constants.PLAN_JETPACK_PREMIUM,
```

Comment:
Noting that I find something confusing here: we're clicking the button inside the "Pro" plan, but we get the "premium" slug. This feels a bit confusing to me - is it intended? Will we know what we're looking at when we look at the events later?

Reply:
I was sure that if it lives inside `[ constants.FEATURE_ALL_PREMIUM_FEATURES_JETPACK ]` object then its slug should be `PLAN_JETPACK_PREMIUM`. Those current and legacy plan names may be quite misleading.

Reply:
I agree, it misled me last time when I was testing 😉